wrangler.toml

```diff
@@ -0,0 +1,11 @@
+name = "firefox-cro-redirector-dev"
+type = "javascript"
+workers_dev = false
+
+[env.staging]
+name = "firefox-cro-redirector-stage"
+routes = [ "https://www.allizom.org/en-US/firefox/*" ]
```

Comment:
Should these be empty by default in the repo? They get populated when `get_data.js` is run, so might folks feel compelled to make changes to these if they are also hard coded in git?